pkg/autoscaler/autoscaler.go

```diff
@@ -142,9 +141,16 @@ func (agg *totalAggregation) observedConcurrencyPerPod(now time.Time) float64 {
 		}
 	}
 	if accumulatedConcurrency == 0.0 {
```

Comment:
do you want o use same approximate zero (1e-8) value here for comparison?

Reply:
Done